CMakeLists.txt

```diff
@@ -61,7 +62,7 @@ etn_configure_file(
     conf/53_assets.xml.in
 
     TARGET      ${PROJECT_NAME}
-    DESTINATION /etc/tntnet/bios.d
+    DESTINATION ${CMAKE_INSTALL_PREFIX}/tntnet/bios.d
```

Comment:
${CMAKE_INSTALL_FULL_SYSCONFDIR}
for /etc, not ${CMAKE_INSTALL_PREFIX} (/usr) ;)

Reply:
fixed. Thx!